src/tests/gov/nasa/jpf/test/java/lang/reflect/MethodTest.java

```diff
@@ -19,6 +19,7 @@
 package gov.nasa.jpf.test.java.lang.reflect;
 
 import gov.nasa.jpf.util.test.TestJPF;
+
```

Comment:
Thanks! Please try to revert changes in spacing (empty) lines that you introduced, so the PR can also be merged on other branches if needed.